crates/conductor_lib/src/conductor/admin.rs

```diff
@@ -201,13 +201,14 @@ impl ConductorAdmin for Conductor {
             id: id.to_string(),
             dna: dna_id.to_string(),
             agent: agent_id.to_string(),
-            storage: StorageConfiguration::Pickle {
+            storage: StorageConfiguration::Lmdb {
                 path: storage_path
                     .to_str()
                     .ok_or_else(|| {
                         HolochainError::ConfigError(format!("invalid path {:?}", storage_path))
                     })?
                     .into(),
+                initial_mmap_bytes: None,
```

Comment:
@willemolding, the second parameter was missing. I've added this line. Is this correct?

Reply:
Yep I must have missed that one. Looking good